### PR TITLE
Remove uneeded *py tutorials and edit the tutorials readm

### PR DESCRIPTION
**Pull request by Idan-BenAmi**
## Pull Request Description:
Remove unneeded tutorials and edit the tutorials readme

## Checklist before requesting a review:
- [ ] I set the appropriate labels on the pull request.
- [ ] I have added/updated the release note draft (if necessary).
- [ ] I have updated the documentation to reflect my changes (if necessary).
- [ ] All function and files are well documented. 
- [ ] All function and classes have type hints.
- [ ] There is a licenses in all file.
- [ ] The function and variable names are informative. 
- [ ] I have checked for code duplications.
- [ ] I have added new unittest (if necessary).

### CODE REVIEW

File: tutorials/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+In this section, we will cover more advanced topics related to quantization. 
+This includes fine-tuning PTQ (Post-Training Quantization) configurations, exporting models,
+and exploring advanced compression techniques. 
+These techniques are crucial for optimizing models further and achieving better performance in deployment scenarios.
+- [MCT notebooks](notebooks/MCT_notebooks.md)
+
+## Quantization for Sony-IMX500 deployment
+This section provides a guide on quantizing pre-trained models to meet specific constraints for deployment on the
+processing platform. Our focus will be on quantizing models for deployment on [Sony-IMX500](https://developer.sony.com/imx500/) processing platform. 
+We will cover various tasks and demonstrate the necessary steps to achieve efficient quantization for optimal
```
Comment (ofirgo) [inline tutorials/README.md:22]: This is not a tutorial but an introduction, the future tense here is mistaken.

File: tutorials/notebooks/IMX500_notebooks.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Sony-IMX500 Notebooks
+
+Here we provide examples on quantizing pre-trained models for deployment on Sony-IMX500 processing platform.
+We will cover various tasks and demonstrate the necessary steps to achieve efficient quantization for optimal
+deployment performance.
```
Comment (ofirgo) [inline tutorials/notebooks/IMX500_notebooks.md:5]: highlight that the exported models from these tutorials are ready to be deployed on IMX500! (plug-and-play)

File: tutorials/notebooks/MCT_notebooks.md [hunk 3]
```diff
@@ -1,30 +1,6 @@
...
-## Introduction
-Dive into the Model-Compression-Toolkit (MCT) with our collection of tutorials, covering a wide 
-range of compression techniques for Keras and Pytorch models. We provide
-both Python scripts and interactive Jupyter notebooks for an
-engaging and hands-on experience.
-
-
-## Keras Tutorials
+# MCT Features
+
```
Comment (ofirgo) [inline tutorials/notebooks/MCT_notebooks.md:2]: missing introduction on purpose? Is this my task on a separate PR? (because the rest of the page seems pretty much completed)

File: tutorials/notebooks/MCT_notebooks.md [hunk 4]
```diff
@@ -147,3 +124,14 @@ engaging and hands-on experience.
   
 </details>
 
+
```
Comment (ofirgo) [inline tutorials/notebooks/MCT_notebooks.md:127]: why so many blank lines?

File: tutorials/notebooks/keras/gptq/example_keras_mobilenet_gptq_mixed_precision.py [hunk 5]
```diff
@@ -1,160 +0,0 @@
-# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
```
Comment (ofirgo) [inline tutorials/notebooks/keras/gptq/example_keras_mobilenet_gptq_mixed_precision.py:1]: Don't we want a notebook that replaces this tutorial before removing it?

File: tutorials/notebooks/keras/ptq/example_keras_mobilenet_mixed_precision_lut.py [hunk 6]
```diff
@@ -1,154 +0,0 @@
-# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
```
Comment (ofirgo) [inline tutorials/notebooks/keras/ptq/example_keras_mobilenet_mixed_precision_lut.py:1]: Don't we want a notebook that replaces this tutorial before removing it?

File: tutorials/notebooks/pytorch/ptq/example_pytorch_mobilenet_mixed_precision_lut.py [hunk 7]
```diff
@@ -1,135 +0,0 @@
-# Copyright 2022 Sony Semiconductor Israel, Inc. All rights reserved.
```
Comment (ofirgo) [inline tutorials/notebooks/pytorch/ptq/example_pytorch_mobilenet_mixed_precision_lut.py:1]: Don't we want a notebook that replaces this tutorial before removing it?

File: tutorials/notebooks/pytorch/qat/example_pytorch_qat.py [hunk 8]
```diff
@@ -1,233 +0,0 @@
-# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
```
Comment (ofirgo) [inline tutorials/notebooks/pytorch/qat/example_pytorch_qat.py:1]: Don't we want a notebook that replaces this tutorial before removing it?

**Review by ofirgo — CHANGES_REQUESTED** [review]

**Review by ofirgo — APPROVED** [review]
